File: Matrices/Easy/1672/1672.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize){
    long max = -1;
    for(int i = 0; i < accountsSize; i++){
        long acc = 0;
        int *row = accounts[i];
        for(int j = 0; j < *accountsColSize; j++){
            acc += row[j];
        }
        if(max < acc){
            max = acc;
        }
    }
    return max;
}

File: Matrices/Easy/1672/1672.js
var maximumWealth = function(accounts) {
    let max = -1
    for(let i = 0; i < accounts.length; i++){
        let acc = 0
        for(let j = 0; j < accounts[i].length; j++){
            acc += accounts[i][j]
        }
        if(max < acc){
            max = acc
        }
    }
    return max
};